qadi_multi_perspective.py

```
@@ -0,0 +1,284 @@
+    print("\n🔍 Intent Detection:")
+    print(f"Primary Intent: {intent_result.primary_intent.value.title()} "
+          f"(confidence: {intent_result.confidence:.0%})")
+    
+    if intent_result.keywords_matched:
+        print(f"Keywords Detected: {', '.join(intent_result.keywords_matched[:5])}")
+    
+    print(f"\n📊 Analysis Perspectives: {', '.join(p.value.title() for p in perspectives)}")
+    print("─" * 70)
+
+
+def display_perspective_result(pr: PerspectiveResult, is_primary: bool = False) -> None:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)